packages/wonder-blocks-switch/src/themes/default.ts

```diff
@@ -5,30 +5,28 @@ import {
     spacing,
 } from "@khanacademy/wonder-blocks-tokens";
 
-// The color of the switch is based on the action color.
-const action = semanticColor.action.secondary.progressive;
-
 const theme = {
     color: {
         bg: {
             switch: {
-                off: action.default.border,
-                disabledOff: semanticColor.action.primary.disabled.background,
+                off: semanticColor.core.border.neutral.default,
+                disabledOff: semanticColor.core.border.disabled.strong,
                 // NOTE: Adding this as a primitive token b/c we don't have a
                 // semantic token for this background color
                 activeOff: color.fadedOffBlack64,
-                on: action.default.foreground,
-                disabledOn: action.press.background,
-                activeOn: action.press.foreground,
+                on: semanticColor.core.background.instructive.default,
+                disabledOn: semanticColor.core.border.instructive.subtle,
+                activeOn: semanticColor.core.background.instructive.strong,
             },
             slider: {
                 on: semanticColor.icon.inverse,
                 off: semanticColor.icon.inverse,
```

Comment:
Will `semanticColor.icon` also be replaced by core tokens?

Reply:
yeah, that would probably be the case, but something to work on later.